src/etc/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
+
+RUN echo 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.7 main' | \
+        sudo tee -a /etc/apt/sources.list
+RUN echo 'deb-src http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.7 main' | \
+        sudo tee -a /etc/apt/sources.list
+RUN apt-get update
+RUN apt-get -y install curl make g++ python2.7 git zlib1g-dev libedit-dev
+RUN apt-get -y --force-yes install llvm-3.7-tools
+
+RUN mkdir /build
+WORKDIR /build
```

Comment:
@alexcrichton IIRC, each `RUN` creates a new file system layer. Combining the `RUN`s into one (just using `&&`) might speed up your `docker build`.

If this image is cached anywhere (which might make sense), you might also want append some clean up calls to the `RUN` calling `apt-get` to reduce the size. I'm no authority on this, but I've often seen stuff like `apt-get autoremove -y && apt-get clean all && rm -rf /var/lib/apt/lists/*`.